langchain4j/src/main/java/dev/langchain4j/service/ServiceOutputParser.java

```diff
@@ -130,6 +130,7 @@ private static String jsonStructure(Class<?> structured, Set<Class<?>> visited)
             }
             jsonSchema.append(format("\"%s\": (%s),\n", name, descriptionFor(field, visited)));
         }
+        jsonSchema.delete(jsonSchema.lastIndexOf(","), jsonSchema.lastIndexOf(",")+1);
```

Comment:
Great one-liner fix. We could consider doing this cleanly with a .join() but that would require switching to streams so this is fine in my opinion.

Reply:
This is causing following exception while trying to create json structure for classes/interfaces without declared fields: 

`Range [-1, 0) out of bounds for length 2. java.lang.RuntimeException: Range [-1, 0) out of bounds for length 2`

Reply:
Fixed with https://github.com/langchain4j/langchain4j/pull/975